tools/abel/validators/README.md

```diff
@@ -0,0 +1,196 @@
+# ABEL Spreadsheet
+
+### Overview
+The ABEL Spreadsheet serves as a user interface for the ABEL application. The
+spreadsheet is the user-friendly interface that allows users to create a
+[Building Configuration file](../../../ontology/docs/building_config.md) from scratch.
+
+## Google Sheets template
+
+The format of an ABEL spreadsheet is well-defined, and it is suggested that you use the following [Google Sheets Template](https://docs.google.com/spreadsheets/d/1tcLjFnHiXUT-xh5C1hRKiUVaUH_CzgSI8zFQ_B8q7vs/copy#gid=980240783)
```

Comment:
nit maybe instead of "The format of an ABEL spreadsheet is well-defined", something like "ABEL spreadsheets have a strict format that must be adhered to"? The latter says "hey you probably should use this template, it has all the columns that ABEL needs", vs the former almost feels like bragging in a sense ("our format is so good and defined")

Reply:
Used "The [ABEL Spreadsheet template](link) has a strict format that must be adhered to"